src/Symfony/Component/Console/Tests/Command/CommandTest.php

```diff
@@ -192,7 +192,8 @@ public function testGetSynopsis()
         $command = new \TestCommand();
         $command->addOption('foo');
         $command->addArgument('bar');
-        $this->assertEquals('namespace:name [--foo] [--] [<bar>]', $command->getSynopsis(), '->getSynopsis() returns the synopsis');
+        $command->addArgument('info');
+        $this->assertEquals('namespace:name [--foo] [--] [<bar> [<info>]]', $command->getSynopsis(), '->getSynopsis() returns the synopsis');
```

Comment:
This was already working before the fixed output escaping in `finishRenderThrowableOrException()`. I just added it to make sure the problem is not located in `getSynopsis()`.